build.gradle

```diff
@@ -48,7 +48,7 @@ apply from: 'build-tools/esplugin-coverage.gradle'
 check.dependsOn jacocoTestReport
 
 esplugin {
-    name 'opendistro-index-state-management'
```

Comment:
Mix of _ and - is weird, are you seeing this being used somewhere else like this?